otherlibs/systhreads/st_win32.h

```diff
@@ -24,7 +24,242 @@ Caml_inline void st_msleep(int msec)
   Sleep(msec);
 }
 
-#include "st_pthreads.h"
+/* POSIX thread implementation of the "st" interface */
```

Comment:
Is this comment still accurate?

Reply:
In this first commit, things are simply moved around, it wouldn't make sense to also change the contents at this point.

Reply:
The first commit is supposed to help reviewers by simplifying the diff from pthreads. I can remove this comment in the latter commits.